crates/store/src/limiter.rs

```diff
@@ -0,0 +1,56 @@
+#[derive(Debug, thiserror::Error)]
+#[error("Parameter {which} exceeded limit {limit}: {size}")]
```

Comment:
Nit: lowercase errors compose better in error reports

```suggestion
#[error("parameter {which} exceeded limit {limit}: {size}")]
```

Reply:
https://github.com/0xMiden/miden-base/discussions/966